src/components/overview/cart.js

```diff
@@ -49,26 +46,70 @@ export default function Cart() {
     setSelected(e.target.value);
   }
 
+  //Change current first SKU based on size
+  function setSkuOnFirstChange(e) {
+    var skus = Object.keys(stateVal.currentStyle.skus);
+    setSku(parseInt(skus[e.target.value]));
+    setFirstSelected(e.target.value);
+  }
+
   //Add to cart
   function addToCart(e) {
     e.preventDefault();
     if (!sku || !quantity) {
-      alert('Please select size!')
-    } else {    axios
-      .post("http://localhost:3000/cart", {
-        sku_id: sku,
-      })
-      .then((response) => {
-        console.log("Added to cart!");
-      })
-      .catch((err) => {
-        console.error(err);
-      });}
+      alert('Please select size!');
+    } else {
+      axios
+        .post('http://localhost:3000/cart', {
+          sku_id: sku,
+        })
+        .then((response) => {
+          console.log('Added to cart!');
+        })
+        .catch((err) => {
+          console.error(err);
+        });
+    }
   }
 
+  const cartStyle = {
```

Comment:
To maintain consistency, we might want to create a class and edit the display in the .css file.